src/js/msp/MSPHelper.js

```diff
@@ -1222,7 +1224,7 @@ MspHelper.prototype.crunch = function(code) {
             if (semver.gte(CONFIG.apiVersion, "1.37.0")) {
                 buffer.push8(BEEPER_CONFIG.dshotBeaconTone);
             }
-            if (semver.gte(CONFIG.apiVersion, "1.39.0")) {
+            if (semver.gte(CONFIG.apiVersion, "1.39.0") && BEEPER_CONFIG.dshotBeaconConditions.getMask() != 0) {
```

Comment:
This won't work. API 1.39 expects this value to be sent, and if it's not sent the firmware will read whatever is sent after it and use it. Also, a mask of `0` is valid.